common/src/types/v0/message_bus/nexus.rs

```diff
@@ -125,6 +125,7 @@ impl From<i32> for NexusShareProtocol {
     fn from(src: i32) -> Self {
         match src {
             1 => Self::Nvmf,
+            0 => Self::Nvmf,
```

Comment:
Nit: Could we put this first so the match is in ascending order starting from 0? (Alternatively you could do `0 | 1 => Self::Nvmf,`)